fake_switches/http_service.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Inap.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from twisted.web.server import Site
+
+
+class SwitchHttpService(object):
+    def __init__(self, ip, port=80, switch_core=None, users=None):
```

Comment:
I have refactored the transport to fix the port discrepancy.
I suggest you rebase your change to use it with the BaseTransport class.